languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -1049,6 +1049,22 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_HOMONYMES_PARONYMES" name="Confusion d’homonymes et paronymes">
+      <rule id="FEUS_OU_FEUX" name="feus ou feux">
+          <antipattern>
+              <token regexp="yes">feu|feus</token>
+              <token postag="[NZ].*" postag_regexp="yes"/>
```

Comment:
I'd also put the `UNKNOWN` tag and tokens that are all-caps (those cases are rare and FPs are probable), see rule editor results

Reply:
I have put the UNKNOWN tag but I didn't put the all-caps as I didn't see FPs in the rule editor. It would also exclude sentences like "Il a des feux AR." and "Il a des feux AV." which are often used to refer to "avant" and "arrière". I think the rule is ready.